autocoder/ai.py

```diff
@@ -20,3 +22,7 @@ def __init__(self, model: BaseChatModel):
     def call(self, messages: List[BaseMessage]) -> str:
         print_system_msg(f"Calling AI with prompt:\n{pprint_messages(messages)}")
         return self.model(messages).content
+
+    def stream(self, messages: List[BaseMessage]) -> Iterator[BaseMessageChunk]:
```

Comment:
@devbuddytest let's return `Iterator[str]` from this method to align with `call` signature. You can extract the body of the chunk with `BaseMessageChunk.content`